nixos/modules/hardware/all-firmware.nix

```diff
@@ -31,7 +31,7 @@ in {
       type = types.bool;
       description = ''
         Turn on this option if you want to enable all the firmware with a license allowing redistribution.
-        (i.e. free firmware and <literal>firmware-linux-nonfree</literal>)
+        (i.e. free and non-free firmware contained in <literal>linux-firmware</literal>)
```

Comment:
Hmm, this sounds confusing, as this option will enable not *just* the (free and non-free) firmware from `linux-firmware`, but also some others (`intel2200BGFirmware`, `rtl8192su-firmware`, `rt5677-firmware`, ...).

I don't feel strongly about this, so feel free to ignore, but I still think just removing this line would be clearest ;)

Reply:
Done :)